### DISCUSSION TITLE
Feedback Wanted: Updating data retention from 90 days to 30 days.

### DISCUSSION THREAD

**Discussion by aspeckofdustin** [community/community#137289] [Other Feature Feedback, Questions, & Ideas]
Hello everyone,

We are planning to schedule an upcoming change that will reduce how far back users can query events data, from 90 days to 30.

This change is being introduced to support better data hygiene, improve performance specifically around utilizing push events, as well as support a shift in how we believe most users engage with this feature.

Currently, you are able to query back up to 90 days worth of data events with schemas you have access to. 
This change will limit the following areas of the platform to the past 30 days:

- Events API
- Atom feed
- /timeline
- /dashboard-feed

We want to hear from our users on how this will directly impact you and your team. 
Specifically, we’d love to gather feedback on the following:

- How often do you use the data beyond 30 days to create push events?
- What concerns or impacts do you foresee when this change will be implemented?
- Do you rely on the 90-day lookback for specific use cases? If so, what are they?

We value your input and want to make sure any changes we implement consider the needs of our users.

Please feel free to share any feedback, concerns, or suggestions you have in the thread below.

**Comment by VasanthGN** [comment]
oo

**Reply by jakebrand24** [reply]
> oo

ok

**Reply by jakebrand24** [reply]
> > oo
> 
> ok

hi

**Reply by jakebrand24** [reply]
> > oo
> 
> ok

how are you

**Reply by jakebrand24** [reply]
> > oo
> 
> ok

im fine

**Reply by jakebrand24** [reply]
> > oo
> 
> ok

tks you

**Reply by jakebrand24** [reply]
> oo

im alone

**Reply by jakebrand24** [reply]
> > oo
> 
> ok

too

**Reply by jakebrand24** [reply]
> > oo
> 
> ok

lol

**Comment by IvankaKHan** [comment]
Hi

**Comment by JorenVsyntra** [comment]
Hey everyone!

**Comment by Cango67** [comment]
``

**Comment by ndquanghung** [comment]
I need auto clean trunk after 30 days using

**Comment by shinybrightstar** [comment]
Hi Community 🌟

Thank you for your posts and comments regarding the recent bug with the Feed sorting in reverse order. Also, data retention has been updated from 90 days to 30 days. Our Feed team released an update on January 30th, 2025, to fix these issues.
Also, we’re working on resolving the “releases not showing” problem, which will be addressed by the end of February 2025.

If you’re still experiencing issues or have more details to share about your experience, please comment directly in this thread so we can monitor the changes effectively.

You can find more info [here](https://github.blog/changelog/2024-11-08-upcoming-changes-to-data-retention-for-events-api-atom-feed-timeline-and-dashboard-feed-features/)

We appreciate you all for helping us continuously improve our platform!
